### ISSUE TITLE
Include metadata in FieldInfo?

### ISSUE THREAD

**Issue by kevinresol** [haxetink/tink_typecrawler#4]
Is it omitted by intention or not?

**Comment by back2dos** [comment]
It is, in that I had no use for it and I'm a YAGNI kinda guy :D 

But if you have found a use case, please feel free to add it!

**Comment by kevinresol** [comment]
The meta usage is described in #5. And I will create a PR for this issue soon, I checked and it is easy.

example: Just like enum as in tink_json

``` haxe
enum Item {
  @:json({ type: 'sword' }) Sword(damage:Int);
  @:json({ type: 'shield' }) Shield(armor:Int);
}
```

But I would like to work with anon struct:

``` haxe
typedef Item = {
  @:someMeta var someField:SomeType;
}
```

**Comment by kevinresol** [comment]
Closed by #9
